prow/plugins/trigger/push.go

```diff
@@ -17,12 +17,44 @@ limitations under the License.
 package trigger
 
 import (
+	"fmt"
+	"strings"
+
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plank"
 )
 
+var mergeRe = `Automatic merge from submit-queue`
+
+func notifyOnSlackIfManualMerge(sc SlackClient, pe github.PushEvent) error {
+	if sc == nil {
+		return nil
+	}
+	sendMessage := false
+	for _, commit := range pe.Commits {
+		if strings.Contains(commit.Message, mergeRe) {
+			sendMessage = true
+			break
+		}
+	}
+	if (pe.Pusher.Name != "k8s-merge-robot") && (pe.Sender.Login != "k8s-merge-robot") {
+		sendMessage = true
+	}
+	if sendMessage {
+		message := fmt.Sprintf("Warning: %s manually merged %s", pe.Repo.Owner.Login, pe.Repo.HTMLURL)
```

Comment:
Can you link directly to the manually merged code, instead of the repo? The "compare" URL looks promising.